packages/slate/src/interfaces/editor.ts

```diff
@@ -1505,21 +1505,32 @@ export const Editor: EditorInterface = {
   ): Range {
     const { voids = false } = options
     let [start, end] = Range.edges(range)
+    const startPointAtEndOfNode = Editor.isEnd(editor, start, start.path);
+    const endPointAtStartOfNode = Editor.isStart(editor, end, end.path);
```

Comment:
Small nitpick: Keep styling consistent with Eslint. If you use VSCode, you could maybe use an Eslint plugin to autoformat on save.
```suggestion
    const startPointAtEndOfNode = Editor.isEnd(editor, start, start.path)
    const endPointAtStartOfNode = Editor.isStart(editor, end, end.path)
```

Reply:
got it.. will correct that in next commit.